MobileWeather/ImageProcessor.m

```diff
@@ -31,9 +31,13 @@ - (instancetype)init {
     return self;
 }
 
-- (UIImage *)imageFromConditionImage:(NSString *)conditionImage {
-    UIImage *image = [UIImage imageNamed:conditionImage];
-    
+- (SDLArtwork *)artworkFromConditionImage:(NSString *)conditionImage imageSize:(ImageSize)imageSize {
+    return [SDLArtwork artworkWithImage:[[self imageFromConditionImage:conditionImage imageSize:imageSize] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate] asImageFormat:SDLArtworkImageFormatPNG];
+}
+
+- (UIImage *)imageFromConditionImage:(NSString *)conditionImage imageSize:(ImageSize)imageSize {
+    UIImage *image = [UIImage systemImageNamed:[self mw_systemImageFromAssetsImage:conditionImage] withConfiguration:[UIImageSymbolConfiguration configurationWithPointSize:[ImageSizeHelper floatForImageSize:imageSize] weight:UIImageSymbolWeightBold]];
```

Comment:
This should probably include being a template image instead of the SDLArtwork method?